### DISCUSSION TITLE
bun install packages slow?

### DISCUSSION THREAD

**Discussion by yyong008** [oven-sh/bun#10053] [Q&A]
The "bun install" command may not be fast in installing packages on Windows, and it could be related to the network. Additionally, there may be other issues encountered. Sometimes the initial installation process may get stuck during resolving.

**Comment by donywan** [comment]
Me too.

**Comment by AQian0** [comment]
Please use a web proxy (but it may not work) or a mirror site (worked for me)

**Reply by donywan** [reply]
> Please use a web proxy (but it may not work) or a mirror site (worked for me)

I using `.bunfig.toml` but still slowly.

```
[install]

# where `bun install --global` installs packages
globalDir = "d:\\.bun\\install\\global"

# where globally-installed package bins are linked
globalBinDir = "d:\\.bun\\bin"

# set default registry as a string
registry = "https://registry.npmmirror.com/"

[install.cache]

# the directory to use for the cache
dir = "d:\\.bun\\install\\cache"
```

**Reply by AQian0** [reply]
This looks a little strange ......

I only set in .bunfig.toml

`registry = “http://registry.npmmirror.com/”`

Maybe it's the https? I'm not sure.
You could also investigate if .bunfig.toml is taking effect correctly.
